### ISSUE TITLE
Range slider does not render properly on Windows Edge browser

### ISSUE THREAD

**Issue by robhatfield** [Shopify/polaris-react#2905]
## Issue summary

The range slider does not render as a circle on Edge.

## Expected behavior

It should look like Chrome or FireFox:
![image](https://user-images.githubusercontent.com/17050937/78719293-b3f74680-78f1-11ea-851c-c6e06b1c086a.png)

## Actual behavior
![image](https://user-images.githubusercontent.com/17050937/78719310-ba85be00-78f1-11ea-8de2-59862f6689e2.png)

## Steps to reproduce the problem

1. Using Edge, go to https://polaris.shopify.com/components/forms/range-slider
2. Observe the circle.

## Reduced test case

The best way to get your bug fixed is to provide a reduced test case. This [CodeSandbox template](https://codesandbox.io/s/q82mlq0m26) is a great starting point.

I don't think I can reduce it any more, the issue is visible in the sample.

## Specifications

- Are you using the React components? (Y/N): Y
- Polaris version number: unsure, it's the latest on polaris.shopify.com
- Browser: Edge 44.18362.449.0 / EdgeHTML 18.18363
- Device: PC
- Operating System: Windows 10

Or run `npx envinfo --system --binaries --browsers --npmPackages react,react-dom,@shopify/polaris` to provide specifications on your environment including version numbers, browser, device, and operating system.

Paste the results here:

```bash

```

**Comment by kyledurand** [comment]
Could you add the version of edge? It's important to differentiate between edge and chromium edge

**Comment by robhatfield** [comment]
Looks like I am still running the legacy edge browser.  I'll check my updates.

**Comment by robhatfield** [comment]
Yesterday I made sure my Windows 10 PC installed all Microsoft updates. The legacy edge browser is still the OS default.  I reproduced the problem and updated the version number in the issue description.

The Shopify help page says that we support the last two versions of every browser.  Does that include legacy edge?  I would say yes.

https://help.shopify.com/en/manual/intro-to-shopify/shopify-admin/supported-browsers

Just to confirm, the slider does render correctly with the Chromium version of Edge.
